votingpool/db.go

```diff
@@ -70,7 +139,7 @@ func putPool(tx walletdb.Tx, votingPoolID []byte) error {
 // loadAllSeries returns a map of all the series stored inside a voting pool
 // bucket, keyed by id.
 func loadAllSeries(tx walletdb.Tx, votingPoolID []byte) (map[uint32]*dbSeriesRow, error) {
```

Comment:
The only downside I can think of is that GetSeries() would then require the manager to be unlocked. I've added a FIXME here too, anyway